sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/models/VariantReference.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.feature.management.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class VariantReference {
+
+    private String name;
+
+    private Object configurationValue;
+
+    private String configurationReference;
```

Comment:
Configuration reference is no longer supported and should be removed here. 
https://github.com/microsoft/FeatureManagement-Python/pull/37